gandlf_splitCSV

```diff
@@ -0,0 +1,66 @@
+#!usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os, argparse, sys, yaml
+from GANDLF.cli import copyrightMessage, split_data_and_save_csvs
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog="GANDLF_SplitCSV",
+        formatter_class=argparse.RawTextHelpFormatter,
+        description="Split the data into training, validation, and testing sets and save them as csvs in the output directory.\n\n"
+        + copyrightMessage,
+    )
+    parser.add_argument(
+        "-i",
+        "--inputCSV",
+        metavar="",
+        default=None,
+        type=str,
+        required=True,
+        help="Input CSV file which contains the data to be split.",
+    )
+    parser.add_argument(
+        "-c",
+        "--config",
+        metavar="",
+        default=None,
+        required=True,
```

Comment:
Hi @scap3yvt ! I have a question here. From one side, config arg is marked as required, while from other side there is a default value if path is empty: https://github.com/mlcommons/GaNDLF/pull/833/files#diff-2512779acde5ad2353679bb5e6eb44f3dfe6e97043026a2e7c12ee36a9358840R53

I am implementing new cli tool now, thus have a question. Which strategy is better in your mind for this command, keep config required or make it optional with default value? I'd fix the behavior in my branch

Reply:
Ok, thanks!